src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Controllers/MemberProfiles/GetMemberProfileWithPreferencesTests.cs

```diff
@@ -18,24 +20,26 @@
 namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Controllers.MemberProfiles;
 public class GetMemberProfileWithPreferencesTests
 {
-    static readonly List<MemberPreference> memberPreferences = new()
-        {
-            new MemberPreference{ PreferenceId = 1, Value =  true },
-            new MemberPreference{ PreferenceId = 2, Value =  true },
-            new MemberPreference{ PreferenceId = 4, Value =  false },
-        };
+
 
     [Test]
-    [MoqInlineAutoData(true, true)]
-    [MoqInlineAutoData(true, false)]
-    [MoqInlineAutoData(false, true)]
-    [MoqInlineAutoData(false, false)]
+    [MoqInlineAutoData(true, true, MemberUserType.Apprentice)]
+    [MoqInlineAutoData(true, false, MemberUserType.Apprentice)]
+    [MoqInlineAutoData(false, true, MemberUserType.Apprentice)]
+    [MoqInlineAutoData(false, false, MemberUserType.Apprentice)]
+    [MoqInlineAutoData(true, true, MemberUserType.Employer)]
+    [MoqInlineAutoData(true, false, MemberUserType.Employer)]
+    [MoqInlineAutoData(false, true, MemberUserType.Employer)]
+    [MoqInlineAutoData(false, false, MemberUserType.Employer)]
     public async Task When_MediatorCommandSuccessful_Then_ReturnOk(
```

Comment:
This test coverage is not sufficient. There are three if conditions in the controller. There should be tests on both sides of these if conditions.

Reply:
I have covered all the three conditions here so not sure what else I need to cover